lang/cs/Org.Apache.REEF.IMRU/OnREEF/Parameters/SerializedPerMapConfiguration.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.IMRU.OnREEF.Parameters
+{
+    [NamedParameter("The per mapper serialized configuration.")]
+    internal sealed class SerializedPerMapConfiguration : Name<string>
```

Comment:
ohh yes...this was needed earlier..not now

On Fri, Aug 28, 2015 at 10:41 AM, Markus Weimer <notifications@github.com>
wrote:

> In
> lang/cs/Org.Apache.REEF.IMRU/OnREEF/Parameters/SerializedPerMapConfiguration.cs
> <https://github.com/apache/incubator-reef/pull/417#discussion_r38224715>:
>
> > +//
> > +//   http://www.apache.org/licenses/LICENSE-2.0
> > +//
> > +// Unless required by applicable law or agreed to in writing,
> > +// software distributed under the License is distributed on an
> > +// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
> > +// KIND, either express or implied.  See the License for the
> > +// specific language governing permissions and limitations
> > +// under the License.
> > +
> > +using Org.Apache.REEF.Tang.Annotations;
> > +
> > +namespace Org.Apache.REEF.IMRU.OnREEF.Parameters
> > +{
> > +    [NamedParameter("The per mapper serialized configuration.")]
> > +    internal sealed class SerializedPerMapConfiguration : Name<string>
>
> Why would this have to be serialized? We can just merge the configuration
> for the IPerMapConfiguration instances into the Driver config, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/417/files#r38224715>.
>